src/flamenco/vm/syscall/fd_vm_syscall_util.c

```diff
@@ -51,28 +54,17 @@ fd_vm_syscall_sol_panic( /**/            void *  _vm,
 
      Note: this syscall is not used by the Rust SDK, only by the C SDK.
      Rust transforms `panic!()` into a log, followed by an abort.
-     It's unclear if this syscall actually makes any sense...
-
-     In Agave, this syscall throws SyscallError::Panic(file, line, column),
-     and then later on the runtime logs the error, including file, line column.
-     Since we can't easily propagate properties of an error in C, we do
-     the log here.  The format of this log is a bit tricky, so we limit the
-     log to a simple and realistic case where file_sz<=56 (where 56 comes from
-     internal constrains of our logging implementation).
+     It's unclear if this syscall actually makes any sense... */
+  FD_VM_CU_UPDATE( vm, file_sz );
 
-     TODO: log when file_sz>56 (note that also line & column have variable size!). */
+  /* Validate string */
+  FD_TRANSLATE_STRING( vm, file_vaddr, file_sz );
 
-  FD_VM_CU_UPDATE( vm, file_sz );
-  char const * file = FD_TRANSLATE_STRING( vm, file_vaddr, file_sz );
-
-  /* Max msg_sz: 37 - 6 + 20 + 20 + file_sz = 71 + file_sz <= 127 => file_sz <= 56 */
-  if( file_sz<=56 ) {
-    /* Note: when file_sz==0, file can be undefined. We use printf with "%.*s" to
-             protect against it. */
-    fd_log_collector_printf_dangerous_max_127( vm->instr_ctx,
-      "SBF program Panicked in %.*s at %lu:%lu", file_sz, file, line, column );
-  }
+  /* Disregard the precise log */
```

Comment:
Why do we disregard here?

Reply:
I'm truncating the log, sol_panic can't be used by rust, only by C sdk (who writes Solana Programs in C!?).
It's an unnecessary effort to make panic work like in agave.